profile.html

```diff
@@ -0,0 +1,220 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="/styles/index.css" />
+    <script src="https://use.fontawesome.com/350296bd22.js"></script>
+    <title>Your Profile</title>
+  </head>
+
+  <body class="body--profile">
+    <header class="header header--profile px-1-5rem">
+        
+        <div class="icon-container">
+          <i class="icon-container__icon fa fa-envelope"> </i>
+          <span class="icon-container__badge"></span>
+        </div>
+        <div class="icon-container">
+            <i class="icon-container__icon fa fa-bookmark-o" aria-hidden="true"> </i>
+          </div>
+      </header>
+    </header>
```

Comment:
Something went wrong here with two nested headers. Is prettier working?

Reply:
:-D no. just tired. Fixed it. Thx